test/grape_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+require 'grape'
+require 'grape/active_model_serializers'
+
+class ActiveModelSerializers::GrapeTest < Minitest::Test
+  include Rack::Test::Methods
+
+  class GrapeTest < Grape::API
+    format :json
+    include Grape::ActiveModelSerializers
+
+    resources :grape do
+      get '/render' do
+        render ARModels::Post.new(title: 'Dummy Title', body: 'Lorem Ipsum')
+      end
+
+      get '/render_with_json_api' do
+        post = ARModels::Post.new(title: 'Dummy Title', body: 'Lorem Ipsum')
+        render(
+          post,
+          meta: { page: 1, total_pages: 2 },
+          adapter: :json_api
+        )
```

Comment:
Nice! :100: 

Is the `(` and `)` around render in these two routes but not in the one-liner a grape convention?